modules/apps/29-fee/keeper/escrow.go

```diff
@@ -129,11 +134,16 @@ func (k Keeper) DistributePacketFeesOnTimeout(ctx sdk.Context, timeoutRelayer sd
 		k.distributePacketFeeOnTimeout(cacheCtx, refundAddr, timeoutRelayer, packetFee)
 	}
 
+	// removes the fee from the store as fee is now paid
+	k.DeleteFeesInEscrow(ctx, packetID)
```

Comment:
Should we update the tests to check that the fee's have been deleted?

Reply:
Yes, i added it.